docs/src/pages/en/components/media-mute-button.mdx

```diff
@@ -61,6 +62,18 @@ Here's an example of how you can replace the volume levels with the words that c
 </media-controller>`}
 />
 
+Alternatively, if you would like to represent all of the volume levels using a single element you could use the `icon` slot instead.
```

Comment:
suggestion: Let's add a fully working sandpack example for this with animation.

Reply:
have pushed this up